src/main/resources/db/migration/V1_293__ORDER_PURPOSES.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE ORDER_PURPOSES (
```

Comment:
I can't see this used anywhere - coming soon?

Reply:
yes - the next one.  I should have mentioned that